package-system/python/darwin_x64/open3d_python.patch

```diff
@@ -1,201 +1,22 @@
 diff --git a/Mac/BuildScript/build-installer.py b/Mac/BuildScript/build-installer.py
-index 4fab488..2f1bd81 100755
+index 4308a20..0217774 100755
 --- a/Mac/BuildScript/build-installer.py
 +++ b/Mac/BuildScript/build-installer.py
-@@ -209,9 +209,9 @@ def library_recipes():
-
-     result.extend([
-           dict(
--              name="OpenSSL 1.1.1g",
--              url="https://www.openssl.org/source/openssl-1.1.1g.tar.gz",
--              checksum='76766e98997660138cdaf13a187bd234',
-+              name="OpenSSL 1.1.1m",
-+              url="https://www.openssl.org/source/openssl-1.1.1m.tar.gz",
-+              checksum='8ec70f665c145c3103f6e330f538a9db',
-               buildrecipe=build_universal_openssl,
-               configure=None,
-               install=None,
-@@ -221,9 +221,9 @@ def library_recipes():
-     if internalTk():
+@@ -274,7 +274,7 @@ def library_recipes():
          result.extend([
            dict(
--              name="Tcl 8.6.8",
--              url="ftp://ftp.tcl.tk/pub/tcl//tcl8_6/tcl8.6.8-src.tar.gz",
--              checksum='81656d3367af032e0ae6157eff134f89',
-+              name="Tcl 8.6.11",
-+              url="https://prdownloads.sourceforge.net/tcl/tcl8.6.12-src.tar.gz",
-+              checksum='87ea890821d2221f2ab5157bc5eb885f',
+               name="Tcl %s"%(tcl_tk_ver,),
+-              url="ftp://ftp.tcl.tk/pub/tcl//tcl8_6/tcl%s-src.tar.gz"%(tcl_tk_ver,),
++              url="https://fossies.org/linux/misc/tcl%s-src.tar.gz"%(tcl_tk_ver,),
```

Comment:
This patch is for grabbing Tkinter python for Linux?
But this patch is in a MacOS specific file. Is this actually used?

Reply:
This is a source tarball, and i believe the linux/unix source also supports mac ( http://tcl.tk/software/tcltk/download.html )